lib/hotel.rb

```diff
@@ -0,0 +1,92 @@
+require_relative 'room'
+require_relative 'reservation'
+
+module HotelSystem
+  class Hotel
+    attr_reader :rooms, :reservations
+    attr_accessor :amount_of_rooms
+    
+    def initialize(amount_of_rooms)
+      @amount_of_rooms = amount_of_rooms
+      @rooms = []
```

Comment:
You could consider using create_rooms in the constructor, because this is a starting state of the hotel.